bin/compute-adam-jars.sh

```diff
@@ -22,8 +22,8 @@ ADAM_REPO="$(cd `dirname $0`/..; pwd)"
 
 CLASSPATH=$("$ADAM_REPO"/bin/compute-adam-classpath.sh)
 
-# list of jars to ship with spark; trim off the first and last from the CLASSPATH
+# list of jars to ship with spark; trim off the first from the CLASSPATH
 # TODO: brittle? assumes appassembler always puts the $BASE/etc first and the CLI jar last
```

Comment:
The code now only trims the first one, right? But is that supposed to be the CLI jar?